README.md

```diff
@@ -1,18 +1,35 @@
 # pyopensci.github.io
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors)
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+[![pages-build-deployment](https://github.com/pyOpenSci/pyopensci.github.io/actions/workflows/pages/pages-build-deployment/badge.svg)](https://github.com/pyOpenSci/pyopensci.github.io/actions/workflows/pages/pages-build-deployment)
 
-This is the website for pyOpenSci. Contributions welcome!!
+This repo contains the source code for the [pyOpenSci.org](https://pyopensci.org) website. Contributions are welcome. Thank you [**pyOpenSci contributors**](#contributors-✨)!!
 
-## Development Environment
+## Getting Started
 
-If you want to contribute to the pyopensci website and you wish to test your changes locally prior to submitting a pr,
-do the following.
+Please explore [the site](https://pyopensci.org) and let us know what your think. If you find a bug or have
+a suggestion for a new feature, [open an issue](https://github.com/pyOpenSci/pyopensci.github.io/issues).
 
-1. You will need to have ruby and bundler installed on your machine.
+## Next Steps
+
+If you are interested in contributing to the pyOpenSci website, please read about
+how we work in [**pyOpenSci's organization-wide contributing guide**](https://www.pyopensci.org/governance/CONTRIBUTING.html).
+Also, review the [CONTRIBUTING.md](./CONTRIBUTING.md)
```

Comment:
@willingc this is a wonderul pr - thank you!
I have a question about this line here - the [packaging guide](https://github.com/pyOpenSci/python-package-guide) is a separate repository as is the [peer review guide](https://github.com/pyOpenSci/software-peer-review). They are separate because we built the books using sphinx and wanted to have unique DOI's for each . But the website is a jekyll website (i know jekyll so it just makes it easy for me to customize).

i've gotten feedback that it's confusing for people to know where each chunk of content lives. Should we link to the repos for each of those guidebook sections of the website in this readme? And perhaps modify this sentence "Also, review the..." to just direct people to those repositories?

please let me know what you think. fresh eyes are always valuable for these types of things.

Reply:
I think it would be best to list each one with a link: website, packaging guide, and peer review guide. It was a little confusing for me so let's be explicit 👍🏼

Reply:
@lwasser I updated the wording here and revamped the content in the `CONTRIBUTING.md` file to better explain the different parts of the project: website and guides. Let me know if you want any additional changes.